src/sentry/snuba/discover.py

```diff
@@ -355,6 +355,7 @@ def query(
     referrer=None,
     auto_fields=False,
     use_aggregate_conditions=False,
+    conditions=None,
```

Comment:
Would be good to update the docstring to call out that these conditions are piped directly to snuba without any aliasing or additional escaping being done.